src/Microsoft.AspNetCore.Dispatcher/DispatcherLoggerExtensions.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Dispatcher.Internal
```

Comment:
You can drop the `.Internal` from this. The class is actually-internal so it doesn't really do anything